guides/common/modules/proc_configuring-the-global-smartproxy-remote-execution-setting-in-project.adoc

```diff
@@ -1,5 +1,5 @@
 [id="Configuring_the_Global_{smart-proxy-context}_Remote_Execution_Setting_in_{project-context}_{context}"]
-= Configuring the Global {SmartProxy} Remote Execution Setting in {Project}
+= Configuring the global {SmartProxy} remote execution setting in {Project}
```

Comment:
unsure; see line 5

Reply:
I think lower-case is fine for the title because "global" and "remote execution" are used generically.